addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -25,18 +25,39 @@ class SaleOrder(models.Model):
         string="Customizable PDF Form Fields",
         readonly=False,
     )
+    ignore_default_ids = fields.Many2many(
```

Comment:
Could you explain the purpose of this field? I believe you can implement your feature without it.

Reply:
My issue was that because the `available_product_documents_ids` field is a computed field, it is therefore not stored and recomputed on any modification of the sale order. 
Since I add the default documents every time the `available_product_documents_ids` is changed to ensure it includes new documents, I had the issue that it would also re-add any default document that was previously removed by the user. And in fact this would happen any time a user where to remove a default document from its quote (which is kinda annoying :smile:).

Its the only solution I could find because to know which document I should ignore when adding the new default documents as I can't know the previous state of `available_product_document_ids` to figure it out.